src/app/user/user.component.html

```diff
@@ -1,47 +1,63 @@
-<div>
-  <label class="text-large">{{account?.firstName + ' ' + account?.lastName}}</label>
-</div>
-<div>
-  <label class="text-large" for='communities'>Communities:</label> <input [(ngModel)]='groupIds'
-                                                                          class="form-control narrow" id="communities"
-                                                                          placeholder="Comma separated ids">
-  <br>
-  <label class="text-large" for="gender">Gender:</label>
-  <br>
-  <select [(ngModel)]="gender" class="custom-select narrow" id="gender">
-    <option [ngValue]="1">Female</option>
-    <option [ngValue]="2">Male</option>
-  </select>
-  <br>
-  <label class="text-large" for='city'>City:</label> <input [(ngModel)]='city'
-                                                            class="form-control narrow" id="city">
-  <br>
-  <label class="text-large" for='ageFrom'>Age from:</label> <input [(ngModel)]='ageFrom'
-                                                                   class="form-control narrow" id="ageFrom"
-                                                                   type="number">
+<div class="container">
+  <div>
+    <label class="text-large">{{account?.firstName + ' ' + account?.lastName}}</label>
+  </div>
+  <div>
+    <label class="text-large" for='communities'>Communities:</label> 
+
+    <div class="row row justify-content-end">
```

Comment:
the group section should be extracted to a separate component. I will create a separate task for this, so this isn't a blocker issue for this PR